ocaml/xapi/test_valid_ref_list.ml

```diff
@@ -69,6 +69,29 @@ let test_flat_map =
       assert_equal ["a"; "d_a"; "d"; "d_d"] (Valid_ref_list.flat_map f l)
     )
 
+let test_filter_map =
+  with_vm_list (fun __context ->
+      function
+      | [vm1; vm2; vm3; vm4] as l ->
+        let f vm =
+          let l = Db.VM.get_name_label ~__context ~self:vm in
```

Comment:
Looks correct, but would be more readable with `n` instead of `l` for the name-label.

Reply:
Good idea, done.